echopype/convert/set_groups_azfp.py

```diff
@@ -90,30 +90,43 @@ def set_platform(self) -> xr.Dataset:
             "platform_type": self.ui_param["platform_type"],
             "platform_code_ICES": self.ui_param["platform_code_ICES"],
         }
-        # Save
+        unpacked_data = self.parser_obj.unpacked_data
+        ping_time = self.parser_obj.ping_time
+
         ds = xr.Dataset(
             {
-                var: ([], np.nan, self._varattrs["platform_var_default"][var])
-                for var in [
-                    "MRU_offset_x",
-                    "MRU_offset_y",
-                    "MRU_offset_z",
-                    "MRU_rotation_x",
-                    "MRU_rotation_y",
-                    "MRU_rotation_z",
-                    "position_offset_x",
-                    "position_offset_y",
-                    "position_offset_z",
-                    "transducer_offset_x",
-                    "transducer_offset_y",
-                    "transducer_offset_z",
-                    "vertical_offset",
-                    "water_level",
-                ]
-            }
+                "tilt_x": (["ping_time"], unpacked_data["tilt_x"]),
+                "tilt_y": (["ping_time"], unpacked_data["tilt_y"]),
```

Comment:
Now that these 2 variables are in `Platform`, I guess our decision in #656 for AZFP should be revised, right? If so, I'll change the `time3/AZFP` entry to be using `ping_time` since it is a case consistent with the EK60 case.

Reply:
If the above is true, I think we can merge this and have all the `time1/2/3` changes be in a separate PR, so that things are cleaner.

Reply:
> Now that these 2 variables are in Platform, I guess our decision in https://github.com/OSOceanAcoustics/echopype/issues/656 for AZFP should be revised, right? If so, I'll change the time3/AZFP entry to be using ping_time since it is a case consistent with the EK60 case.

Yes, I think we should revise this #656. Should we change it to `time3` or `time2` for these specific variables? 

> If the above is true, I think we can merge this and have all the time1/2/3 changes be in a separate PR, so that things are cleaner.

Yes, I was going to do a separate PR of the times. That's why I wanted to do this issue first.

Reply:
Sounds good, let's get this merged first. I have revised #656 text -- and you are right, this should be `time2` since it is data related to platform motion and positions.

Reply:
Great, thank you for revising that!